include/openssl/evp.h

```diff
@@ -698,6 +698,16 @@ OPENSSL_EXPORT int EVP_PKEY_derive_set_peer(EVP_PKEY_CTX *ctx, EVP_PKEY *peer);
 OPENSSL_EXPORT int EVP_PKEY_derive(EVP_PKEY_CTX *ctx, uint8_t *key,
                                    size_t *out_key_len);
 
+// |EVP_PKEY_check| validates the key-pair given by |ctx|. If the key type is supported,
```

Comment:
Style required for docs:
```suggestion
// EVP_PKEY_check validates the key-pair given by |ctx|. If the key type is supported,
```